python/PyQt6/core/auto_generated/browser/qgsdataitem.sip.in

```diff
@@ -431,7 +437,7 @@ Ownership of the returned objects is transferred to the caller.
 Refresh the items from a specified list of child items.
 %End
 
-    bool deferredDelete();
+    bool deferredDelete() const;
```

Comment:
This seems odd to be marked const -- is deleting an object const?

Reply:
Eww, that's not nice! Could you amend the documentation to start with "Returns TRUE if the item is scheduled to be deleted". At least that'd make things a bit clearer